dist/bin/current/helperScripts/setupRuntimeEnvironment.sh

```diff
@@ -31,14 +31,44 @@ checkAndSetJDKInFolder ${PWD}/dist
 
 [[ -z $JAVA_HOME ]] && checkAndSetJDKInFolder ~/.roddy
 
-[[ -z $JAVA_HOME ]] && echo "There was no java runtime environment or jdk setup. Roddy cannot be compiled." && exit 1
-[[ -z $GROOVY_HOME ]] && echo "Groovy SDK / Runtime not found, Roddy cannot be compiled or started." && exit 1
+if [[ ! -z $JAVA_HOME && ! -z $GROOVY_HOME ]]; then
+  PATH=$JDK_HOME/bin:$JAVA_HOME/bin:$GROOVY_HOME/bin:$PATH
+#  JFX_LIBINFO_FILE=~/.roddy/jfxlibInfo
+#  if [[ ! -f ${JFX_LIBINFO_FILE} ]] || [[ ! -f `cat ${JFX_LIBINFO_FILE}` ]]; then
+#    echo `find ${JAVA_HOME}/ -name "jfxrt.jar"` > ${JFX_LIBINFO_FILE}
+#  fi
+  [[ -z $JAVA_HOME ]] && echo "There was no java runtime environment or jdk setup. Roddy cannot be compiled." && exit 1
```

Comment:
Did you forget to close the if block? This will never succeed and report, because JAVA_HOME is `!-z`.